src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/Contracts/GraphComparisonRequest.cs

```diff
@@ -0,0 +1,44 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.ExecutionPlan.Contracts
+{
+    public class GetGraphComparisonParams
+    {
+        /// <summary>
+        /// First query plan's XML file text for comparison.
+        /// </summary>
+        public string FirstQueryPlanXmlText { get; set; }
```

Comment:
Are all execution plans going to be in XML format? If we're supporting generic plans then the contract should be made more generic too - @aasimkhan30 you did that for your stuff already right?

Reply:
Yeah, @lewis-sanchez  there is a class ExecutionPlanGraphInfo that you can use.